src/authorities/InstanceAuthority.ts

```diff
@@ -126,6 +151,10 @@ export default class InstanceAuthority {
             this.currentInstances.push(instance); // Add instance to the in-memory data, so it can be called upon rapidly without polling DB
             this.printActives(); // Show currently running alerts in console / log
 
+            // Subscribe to world rabbit queues to listen to messages
+            this.queueAuthority.syncActiveInstances(this.currentInstances);
+            await this.queueAuthority.startQueuesForInstance(instance);
```

Comment:
It was mainly due to the fact we couldn't call instance authoirty from queue authority or we'd be in a dependency loop. Also we want to outright control what the queues think are the current instances absolutely, with no margin for error, so for the sake of 2 extra lines it would be 100% accurate all the time.

Reply:
Fair enough. I would have liked to see an abstraction so the two do not depend on each other using something like an event emitter, for the sake of easier injection. But as these are the only two I do not think it is an issue.